skupstina_django/scraper/scrape_utils_docu.py

```diff
@@ -0,0 +1,53 @@
+import io
+import docx2txt
+from PyPDF2 import PdfFileReader
+from bs4 import BeautifulSoup
+from .scrape_utils_requests import requests_retry_session
+from skupstina_django.settings import ACTS_ROOT_URL as root_url
+
+
+def extract_docxfile_data(url_docx: str) -> str:
+    response = requests_retry_session().get(url_docx)
+    file = io.BytesIO(response.content)
+    return docx2txt.process(file)
+
+
+def extract_pdffile_data(url_pdf: str) -> str:
+    response = requests_retry_session().get(url_pdf)
+    file = io.BytesIO(response.content)
+    pdf_reader = PdfFileReader(file)
+    pdf_raw_data = ''
+    for page in range(pdf_reader.numPages):
+        pdf_raw_data += pdf_reader.getPage(page).extractText() + '\n'
```

Comment:
Wow, you managed to index docx and PDF files? 😮

Reply:
Yeah!! I tested out a few repos and these ones worked best. PyPDF2 is a quite well known one.